### DISCUSSION TITLE
xposed installer

### DISCUSSION THREAD

**Discussion by jangmoon1** [community/community#21057] [Other Feature Feedback, Questions, & Ideas]
![fawaw](https://user-images.githubusercontent.com/93103131/180384645-1e5f719b-bf3b-438a-8bb7-e59915483e49.jpg)

After downloading the xposed installer, the installation screen does not appear on startup. Please let me know if you have a solution.

**Comment by FahdSeddik** [comment]
Hey! @jangmoon1 

I guess you have only installed the apk of xposed framework. You need to flash the zip file of xposed framework on your device in order to successfully install it.

1. Flash the Framework(zip file) on custom recovery.
2. Reboot Sytem.
3. Install Xposed Installer.
4. Install necessary modules.

Hope this helps!

**Reply by aweey666** [reply]
hi, FahdSeddik i have same problems but im in emulator and using LD Player can u help me so solve this problem pls let me know thanks

**Reply by FahdSeddik** [reply]
In this case, I found this video really helpful 
[link](https://youtu.be/99tCBz0A-R0).

**Reply by 3IMAD69** [reply]
> In this case, I found this video really helpful [link](https://youtu.be/99tCBz0A-R0).

DELETED VIDEO , can you explain how to slove it ?
